packages/@lwc/engine-core/src/framework/rendering.ts

```diff
@@ -172,20 +174,49 @@ function mountElement(vnode: VElement, parent: ParentNode, anchor: Node | null)
     const {
         sel,
         owner,
-        data: { svg },
+        data: { svg, isStatic },
     } = vnode;
 
+    if (isStatic) {
+        if (vnode.protoElm) {
+            const el = (vnode.elm = cloneNode(vnode.protoElm, true));
+
+            linkNodeToShadow(el, owner, true);
+
+            if (process.env.NODE_ENV !== 'production') {
+                // @todo: take out the restrictions patching
+                // @todo: should we traverse the tree to set restrictions on inner nodes?
+                fallbackElmHook(el, vnode);
+            }
+
+            insertNode(el, parent, anchor);
+            return;
+        }
+
+        // @todo: hackalert: for now, let's set the owner in all children
+        const ch = vnode.children;
+        for (let i = 0, n = ch.length; i < n; i++) {
+            ch[i]!.owner = owner;
+        }
+    }
+
     const namespace = isTrue(svg) ? SVG_NAMESPACE : undefined;
     const elm = createElement(sel, namespace);
-    linkNodeToShadow(elm, owner);
 
+    linkNodeToShadow(elm, owner);
     fallbackElmHook(elm, vnode);
+
     vnode.elm = elm;
 
     patchElementPropsAndAttrs(null, vnode);
 
     insertNode(elm, parent, anchor);
     mountVNodes(vnode.children, elm, null);
+
+    if (isStatic) {
+        // clone the rendered node in case it is manually manipulated.
+        vnode.protoElm = cloneNode(elm, true);
```

Comment:
I'm not sure I completely understand the usage of the `vnode.protoElm`, from what I understand it looks like we keep a reference to elm in both `vnode.protoElm` and `vnode.elm` and if we call mountElement against the `vnode` again, we replace the reference to `vnode.elm` with `vnode.protoElm`.  

I'm not super familiar with this area but could do you explain why we clone each time?